package.json

```diff
@@ -15,37 +15,42 @@
   "scripts": {
     "build": "ember build",
     "start": "ember server",
-    "test": "ember try:each"
+    "test": "ember test",
+    "test:all": "ember try:each"
   },
   "dependencies": {
     "broccoli-funnel": "^2.0.1",
-    "broccoli-merge-trees": "^2.0.0",
-    "ember-cli-babel": "^6.11.0",
+    "broccoli-merge-trees": "^3.0.0",
+    "ember-cli-babel": "^6.12.0",
     "pixelmatch": "^4.0.2",
-    "request": "^2.83.0",
+    "request": "^2.87.0",
     "simple-headless-chrome": "^4.3.10"
   },
   "devDependencies": {
-    "broccoli-asset-rev": "^2.4.5",
-    "ember-cli": "~2.18.1",
-    "ember-cli-dependency-checker": "^2.0.0",
+    "broccoli-asset-rev": "^2.7.0",
+    "ember-cli": "~3.1.4",
+    "ember-cli-dependency-checker": "^2.1.1",
     "ember-cli-eslint": "^4.2.3",
     "ember-cli-htmlbars": "^2.0.1",
     "ember-cli-htmlbars-inline-precompile": "^1.0.0",
     "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-qunit": "^4.0.0",
+    "ember-cli-qunit": "^4.3.2",
     "ember-cli-release": "^1.0.0-beta.2",
     "ember-cli-shims": "^1.1.0",
     "ember-cli-sri": "^2.1.0",
-    "ember-cli-uglify": "^2.0.0",
+    "ember-cli-uglify": "^2.1.0",
     "ember-disable-prototype-extensions": "^1.1.2",
     "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^1.0.0",
-    "ember-resolver": "^4.0.0",
-    "ember-source": "~2.18.0",
-    "eslint-plugin-ember": "^5.0.3",
-    "eslint-plugin-node": "^5.2.1",
-    "loader.js": "^4.2.3"
+    "ember-load-initializers": "^1.1.0",
+    "ember-maybe-import-regenerator": "^0.1.6",
+    "ember-resolver": "^4.5.5",
+    "ember-source": "~3.1.0",
+    "ember-source-channel-url": "^1.0.1",
+    "ember-try": "^0.2.23",
+    "eslint": "^4.19.1",
```

Comment:
Please remove this dependency (eslint), as it is will be automatically included by ember-cli-eslint anyhow.

Reply:
I can remove it,
but I think it's safer to list eslint here
instead of relying on other dependencies to install it

your call :)

Reply:
OK, we can also leave it, I guess it doesn't matter too much!